### PR TITLE
Sorr task64 update coding style doc

### PR DESCRIPTION
**Pull request by DanilYachmenev**
#64 

Doing a big big refactoring of a coding style doc

Basically:
- Making it look good and more readable
- Updating with actual info
- Adding missing pieces that were suggested throughout the latest month

The doc is huge so takes time
should finish today/tmrw

### CODE REVIEW

File: docs/Coding_Style_Guide.md [hunk 1]
```diff
@@ -1,162 +1,6 @@
 # Sorrentum - Python Style Guide
```
Comment (gpsaggese) [inline docs/Coding_Style_Guide.md:1]: Looks good. Can you do a round of lint_md.sh to make reflow?

**Comment by DanilYachmenev** [comment]
@samarth9008 pls take a look at the newly updated doc with fresh eyes, let me know if smth is unclear
Focus on the newly added arts and text block structures, do not need to spend too much time here

Just want to understand if everything is more or less clear and that there are no obvious flaws

**Comment by samarth9008** [comment]
@DanilYachmenev Sure, I'll take a look.

**Review by gpsaggese — COMMENTED** [review]
LGTM

**Pull request merged**